float64 p_x
float64 p_y
float64 p_z

float64 vWorld_x
float64 vWorld_y
float64 vWorld_z

float64 vBody_x
float64 vBody_y
float64 vBody_z

float64 rpy_x
float64 rpy_y
float64 rpy_z

float64 omegaBody_x
float64 omegaBody_y
float64 omegaBody_z

float64 omegaWorld_x
float64 omegaWorld_y
float64 omegaWorld_z

float64 quat_0
float64 quat_1
float64 quat_2
float64 quat_3

float64 p1_x
float64 p1_y
float64 p1_z
float64 p2_x
float64 p2_y
float64 p2_z
float64 p3_x
float64 p3_y
float64 p3_z
float64 p4_x
float64 p4_y
float64 p4_z

float64 pz_1
float64 pz_2
float64 pz_3
float64 pz_4
